### ISSUE TITLE
method prepareInterstitial doesn't display ads

### ISSUE THREAD

**Issue by perscrew** [rahadur/capacitor-admob#11]
Hi, I'm trying to display an interstitial ad, and nothing happens on my Android 9.0 smarthphone.

When I'm opening the chrome debugger, I see the capacitor call for your plugin : ```native AdMob.prepareInterstitial (#79027624) ``` but the promise is never rejected or resolved. This is strange. I tried to add a listener with the option ```autoShow: false```, same behaviour it is not called.

Here is some code sample : 

app.component.ts : 
```
if (this.platform.is('android')) {
        AdMob.initialize({ appId: AppSettings.ADMOB_APP_ID });
        AdMob.addListener('onAdLoaded', (info: boolean) => {
          console.log('Interstitial Ad Loaded');
          AdMob.showInterstitial();
        });
 }
```

admob.service.ts
```
 AdMob.prepareInterstitial({ adId: AppSettings.ADMOB_INT_ID, autoShow: false }).then(
            value => {
              console.log('interstitial then ok', value); // true
            },
            error => {
              console.error('interstitial error', error); // show error
            }
 );
```

Another point, I replaced the 'app_id' in the android capacitor admob plugin (string.xml), when I set it, in the main Manifest it raises an error of meta duplication.

Can anybody help ?

**Comment by rahadur** [comment]
Thank you for your report. I am decided that i will  publish a new version of this plugins very soon. i found that google has been changed many api from their Admob SDK.

**Comment by rahadur** [comment]
I think this problem is now fixed on new version 0.0.9.
